index.js

```diff
@@ -0,0 +1,50 @@
+"use strict"
+//Problem 1
+// function iterativeOddSumTo(number){
+//     let sum = 0;
+//     for(let i=1; i<=number; i++){
+//         i%2===0 ? sum+=0 : sum+=i;
```

Comment:
Тернарний оператор тут не потрібен, оскільки ти щось робиш з змінною sum тільки у випадку якщо i%2===0 поверне false

Тут логічніше використати звичайну умовну конструкцію
```suggestion
//        if (i%2!==0) {
//             sum+=i;
//.         };
```